<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterRack</class>
 <widget class="QMainWindow" name="FilterRack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Rack [Raw]</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="FlowWidget" name="filters_flow"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actn_insert_filter"/>
   <addaction name="separator"/>
   <addaction name="actn_select_filter_page"/>
  </widget>
  <action name="actn_insert_filter">
   <property name="icon">
    <iconset resource="../qrc/ic_resources.qrc">
     <normaloff>:/icon/add_filter</normaloff>:/icon/add_filter</iconset>
   </property>
   <property name="text">
    <string>Insert Filter</string>
   </property>
   <property name="toolTip">
    <string>Load and insert a new Filter Plugin to the end of the current Filter Page</string>
   </property>
  </action>
  <action name="actn_next_page">
   <property name="icon">
    <iconset resource="../qrc/ic_resources.qrc">
     <normaloff>:/icon/arrow_right</normaloff>:/icon/arrow_right</iconset>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
   <property name="toolTip">
    <string>Go to the next Filter Page</string>
   </property>
  </action>
  <action name="actn_previous_page">
   <property name="icon">
    <iconset resource="../qrc/ic_resources.qrc">
     <normaloff>:/icon/arrow_left</normaloff>:/icon/arrow_left</iconset>
   </property>
   <property name="text">
    <string>Previous Page</string>
   </property>
   <property name="toolTip">
    <string>Go to the previous Filter Page</string>
   </property>
  </action>
  <action name="actn_select_filter_page">
   <property name="icon">
    <iconset resource="../qrc/ic_resources.qrc">
     <normaloff>:/icon/select_filter_page</normaloff>:/icon/select_filter_page</iconset>
   </property>
   <property name="text">
    <string>Select Filter Page</string>
   </property>
   <property name="toolTip">
    <string>Select a Filter Page from the list</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlowWidget</class>
   <extends>QFrame</extends>
   <header>gui.flow_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/ic_resources.qrc"/>
 </resources>
 <connections/>
</ui>
